app/HutchManager/Models/User/UserModel.cs

```diff
@@ -11,10 +11,21 @@ public class UserModel
 
   [EmailAddress] public string? Email { get; set; } 
   
+  public string FullName { get; set; } = string.Empty;
+
+  public bool AccountConfirmed { get; set; }
+
+  public string Id { get; set; } = string.Empty;
+  
   public UserModel(ApplicationUser entity)
   {
-    Username = entity.UserName;
-    Email = entity.Email;
+    {
+      Username = entity.UserName;
+      Email = entity.Email;
+      FullName = entity.FullName;
+      AccountConfirmed = entity.AccountConfirmed;
+      Id = entity.Id;
+    }
```

Comment:
❓  What does embedding in another set of curly brackets do here?

Reply:
It should have been single curly bracket. I will update them correctly.